src/main/java/com/fabriccommunity/thehallow/util/Contributors.java

```diff
@@ -33,7 +33,7 @@
 	
 	private static void initContributors() {
 		ModMetadata metadata = FabricLoader.getInstance().getModContainer(TheHallow.MOD_ID)
-			.orElseThrow(() -> new IllegalStateException("Cannot find spooky time mod, report to fabric loader!"))
+			.orElseThrow(() -> new IllegalStateException("Cannot find The Hallow mod, please report the issue to the authors!"))
```

Comment:
mod name is fixed, but you should still report to loader because this can only happen if loader is bugged